extensions/gdx-freetype/src/com/badlogic/gdx/graphics/g2d/freetype/FreeTypeFontGenerator.java

```diff
@@ -345,6 +345,8 @@ public FreeTypeBitmapFontData generateData (FreeTypeFontParameter parameter, Fre
 			}
 			ownsAtlas = true;
 			packer = new PixmapPacker(size, size, Format.RGBA8888, 1, false, packStrategy);
+			packer.setCurrentTransparentColor(parameter.color);

```

Comment:
Shadow cannot be simply fixed. That will require adding major feature like TextureBleeder (https://github.com/JFixby/TextureBleeder) to the ```FreeTypeFontGenerator```. 

I may suggest gdx-users never to use shadow feature and draw your string twice instead: one pass for the shadow and one for the actual font.